tests/integration/usage_statistics/test_usage_statistics_messages.py

```diff
@@ -438,6 +440,145 @@ def test_generate_messages_with_defaults():
             "ge_version": "0.11.5.manual_testing",
         }
     ],
+    # BaseDataContext.test_yaml_config() MESSAGES
+    "data_context.test_yaml_config": generate_messages_with_defaults(
```

Comment:
Mostly working, will hold off on full review until @anthonyburdi gets in touch

Reply:
@Shinnnyshinshin it looks like the lambdas just needed to warm up and I tested too early. These tests are passing now, and I just re-triggered CI to verify there as well. 🤞